### ISSUE TITLE
Lollypop 0.9.94 not launching 

### ISSUE THREAD

**Issue by rgiyf** [gnumdk/lollypop#552]
-Lollypop version: 0.9.94.1-1ppa1ubuntu1.1
-Gtk Version: 3.16
-Linux distribution: Ubuntu 15.10 64-bit

-Describe your bug here:
Lollypop not launching after latest update. Terminal output:
```
:~$ lollypop
Traceback (most recent call last):
  File "/usr/bin/lollypop", line 25, in <module>
    from lollypop.application import Application
  File "/usr/lib/python3/dist-packages/lollypop/application.py", line 40, in <module>
    from lollypop.window import Window
  File "/usr/lib/python3/dist-packages/lollypop/window.py", line 15, in <module>
    from lollypop.container import Container
  File "/usr/lib/python3/dist-packages/lollypop/container.py", line 22, in <module>
    from lollypop.view_artist import ArtistView
  File "/usr/lib/python3/dist-packages/lollypop/view_artist.py", line 18, in <module>
    from lollypop.pop_info import InfoPopover
  File "/usr/lib/python3/dist-packages/lollypop/pop_info.py", line 21, in <module>
    from lollypop.widgets_web import OpenLink
  File "/usr/lib/python3/dist-packages/lollypop/widgets_web.py", line 14, in <module>
    gi.require_version('WebKit2', '4.0')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 100, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace WebKit2 not available
```

**Comment by gnumdk** [comment]
Fixed in git.
But you can fix this also by installing:
http://packages.ubuntu.com/fr/wily/gir1.2-webkit-3.0

**Comment by rgiyf** [comment]
@gnumdk package already installed. When you will add new version to ppa?

**Comment by gnumdk** [comment]
@rgiyf  And you get this error?

**Comment by gnumdk** [comment]
@rgiyf  gir1.2-webkit2-3.0 ;)

**Comment by rgiyf** [comment]
@gnumdk ok. Installed gir1.2-webkit2-3.0 but still 
```
:~$ lollypop
Traceback (most recent call last):
  File "/usr/bin/lollypop", line 25, in <module>
    from lollypop.application import Application
  File "/usr/lib/python3/dist-packages/lollypop/application.py", line 40, in <module>
    from lollypop.window import Window
  File "/usr/lib/python3/dist-packages/lollypop/window.py", line 15, in <module>
    from lollypop.container import Container
  File "/usr/lib/python3/dist-packages/lollypop/container.py", line 22, in <module>
    from lollypop.view_artist import ArtistView
  File "/usr/lib/python3/dist-packages/lollypop/view_artist.py", line 18, in <module>
    from lollypop.pop_info import InfoPopover
  File "/usr/lib/python3/dist-packages/lollypop/pop_info.py", line 21, in <module>
    from lollypop.widgets_web import OpenLink
  File "/usr/lib/python3/dist-packages/lollypop/widgets_web.py", line 14, in <module>
    gi.require_version('WebKit2', '4.0')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 104, in require_version
    (namespace, version))
ValueError: Namespace WebKit2 not available for version 4.0
```

**Comment by gnumdk** [comment]
https://github.com/quodlibet/quodlibet/issues/1908

**Comment by rgiyf** [comment]
Yep. Now it's working. You know, Lollypop is kind of huge part of life now :)

**Comment by gnumdk** [comment]
And you won lyrics support ;)

**Comment by pawelad** [comment]
Same here - `gir1.2-webkit2-3.0` was installed and I'm getting `ValueError: Namespace WebKit2 not available for version 4.0`.

@rgiyf how did you fix it?

**Comment by rgiyf** [comment]
@pawelad sudo apt-get install gir1.2-webkit2-4.0

**Comment by pawelad** [comment]
@rgiyf unfortunately not available on elementary OS, but thanks anyway.

**Comment by gnumdk** [comment]
WIll be fixed in 0.9.95...
https://plus.google.com/u/0/104349425299854550670/posts/6Y55qKA5DLW
